library/sl

```diff
@@ -8,7 +8,7 @@ Tags: 7, latest
 Architectures: amd64
 GitFetch: refs/heads/7
 Directory: sl7
-GitCommit: 340cfa6c6f8cb3f2b8427bd9271a57d30c3d7d4c
+GitCommit: 69fbf6ff9797b07eb95078df7f581124ac238879
```

Comment:
```suggestion
GitCommit: 6f0afe07c9abcb8952c22ab76ed69d1fc757602e
```

Update to reflect the new commit id in https://github.com/scientificlinux/sl-docker/commits/7